hrmcalcs2oo.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+class hrmcalcs:
+    def __init__(self, timebeat, peakvalues, start_min, end_min):
+        self.timebeat = timebeat
+        self.start_min = start_min
+        self.end_min = end_min
+        self.peakvalues = peakvalues
+        self.hrminstant()
+        self.hrmaverage()
+
+    def hrminstant(self):
+        self.instant_hr = []
+        for i in self.timebeat:
+            a = round((60/i), 1)
+            self.instant_hr.append(a)
+
+
+    def hrmaverage(self):
+        start_time = 60 * self.start_min
```

Comment:
This implies a certain unit... need to be explicit of what is expected / assumed here.